x-pack/plugins/apm/common/runtime_types/agent_configuration/config_setting_definitions/config_setting_definitions.ts

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import * as t from 'io-ts';
+import { AgentName } from '../../../../typings/es_schemas/ui/fields/agent';
+import { transactionSampleRateRt } from '../transaction_sample_rate_rt';
+import { getIntegerRt } from '../integer_rt';
+import { captureBodyRt } from '../capture_body_rt';
+import { BYTE_UNITS, bytesRt } from '../bytes_rt';
+import { DURATION_UNITS, durationRt } from '../duration_rt';
+
+interface BaseSetting {
+  key: string;
+  label: string;
+  defaultValue?: string;
+  helpText: string;
+  placeholder?: string;
+  validation?: t.Type<any, string, unknown>;
+  validationError?: string;
+  includeAgents?: AgentName[];
+  excludeAgents?: AgentName[];
```

Comment:
Is `includeAgents`/`excludeAgents` clear enough? Should it instead be `onlyAgents`/`notAgents`

Reply:
I like `includeAgents`/`excludeAgents`.